src/unaudited/ECDSAServiceManagerBase.sol

```diff
@@ -234,6 +249,20 @@ abstract contract ECDSAServiceManagerBase is
         return restakedStrategies;
     }
 
+    /**
+     * @notice Sets the rewards initiator address.
+     * @param newRewardsInitiator The new rewards initiator address.
+     * @dev Only callable by the owner.
+     */
+    function setRewardsInitiator(address newRewardsInitiator) external onlyOwner {
+        _setRewardsInitiator(newRewardsInitiator);
+    }
+
+    function _setRewardsInitiator(address newRewardsInitiator) internal {
+        emit RewardsInitiatorUpdated(rewardsInitiator, newRewardsInitiator);
+        rewardsInitiator = newRewardsInitiator;
+    }
+
     // storage gap for upgradeability
     // slither-disable-next-line shadowing-state
     uint256[50] private __GAP;
```

Comment:
Need to update the storage gap for the new state

Reply:
by how much?

Reply:
Needs to be 49 now.  

Basically you allocate up front 50 slots so when you make storage changes you don't shift the storage of contracts in the inheritance graph. Then for each storage slot you use in the contract you deduct 1 from your gap